### DISCUSSION TITLE
Is there any way to cleanly install KOreader on Kindle devices without having the kindle OS?

### DISCUSSION THREAD

**Discussion by GMGuimaraes** [koreader/koreader#10693] [General]
My kindle paperwhite 10th generation had a factory problem where its internal memory was corrupted even though the device worked. But when connecting via usb on the desktop to pass books, it occurred as a pen drive with the same problem and windows asked to format it. After that, the device is in an eternal boot loop.
So I'd like to know if there's a way to install KOreader without needing the Jailbreak steps with the existing OS, because unlike a computer where I can reinstall the original system or another, on kindle I didn't find anything that comes even close to that, and searching for solution I found KOreader alternative.

**Comment by NiLuJe** [comment]
Not really, no.
